src/System.Management.Automation/security/SecuritySupport.cs

```diff
@@ -1336,6 +1337,14 @@ public enum ResolutionPurpose
 
     internal static class AmsiUtils
     {
+        static AmsiUtils()
+        {
+#if !UNIX
+            s_amsiInitFailed = !CheckAmsiInit();
+            PSEtwLog.LogAmsiUtilStateEvent("init-" + s_amsiInitFailed.ToString(), s_amsiContext.ToString() + "-" + s_amsiSession.ToString());
```

Comment:
Please use C# string formatting instead of string concatenation.  You can use the new C# syntax.  Also you do not need to explicitly call `ToString()` on the formatting arguments.

```csharp
PSEtwLog.LogAmsiUtilStateEvent($"init-{s_amsiInitFailed}{s_amsiContext}-{s_amsiSession}");
```

Reply:
Thanks done